### PR TITLE
feat(v1.13.2): Updating helm template for 1.13.2

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham@chaosnative.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] DCO signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Review by ksatchit — APPROVED** [review]

File: charts/litmus-2-0-0-beta/values.yaml [hunk 1]
```diff
@@ -57,18 +57,18 @@ portal:
...
         pullPolicy: "Always"
       containerPort: 8080
       env:
-        SUBSCRIBER_IMAGE: "litmuschaos/litmusportal-subscriber:1.13.0"
-        EVENT_TRACKER_IMAGE: "litmuschaos/litmusportal-event-tracker:1.13.0"
+        SUBSCRIBER_IMAGE: "litmuschaos/litmusportal-subscriber:2.0.0-Beta1"
+        EVENT_TRACKER_IMAGE: "litmuschaos/litmusportal-event-tracker:2.0.0-Beta1"
         ARGO_SERVER_IMAGE: "argoproj/argocli:v2.9.3"
-        ARGO_WORKFLOW_CONTROLLER_IMAGE: "argoproj/workflow-controller:v2.9.3"
+        ARGO_WORKFLOW_CONTROLLER_IMAGE: "argoproj/workflow-controller:2.0.0-Beta1"
```
Comment (rajdas98) [inline charts/litmus-2-0-0-beta/values.yaml:67]: Not required to change here

**Comment by rajdas98** [comment]
Also, we need to change the version here- https://github.com/litmuschaos/litmus-helm/blob/f6409fda6e4b08e18225f00bdb06f92825a596a9/charts/litmus-2-0-0-beta/values.yaml#L23

File: charts/litmus-2-0-0-beta/README.md [hunk 2]
```diff
@@ -58,18 +58,18 @@ A Helm chart to install litmus portal
...
 | portal.server.authServer.resources | object | `{}` |  |
 | portal.server.graphqlServer.containerPort | int | `8080` |  |
 | portal.server.graphqlServer.env.ARGO_SERVER_IMAGE | string | `"argoproj/argocli:v2.9.3"` |  |
 | portal.server.graphqlServer.env.ARGO_WORKFLOW_CONTROLLER_IMAGE | string | `"argoproj/workflow-controller:v2.9.3"` |  |
 | portal.server.graphqlServer.env.ARGO_WORKFLOW_EXECUTOR_IMAGE | string | `"argoproj/argoexec:v2.9.3"` |  |
-| portal.server.graphqlServer.env.LITMUS_CHAOS_OPERATOR_IMAGE | string | `"litmuschaos/chaos-operator:1.13.0"` |  |
-| portal.server.graphqlServer.env.LITMUS_CHAOS_RUNNER_IMAGE | string | `"litmuschaos/chaos-runner:1.13.0"` |  |
-| portal.server.graphqlServer.env.SUBSCRIBER_IMAGE | string | `"litmuschaos/litmusportal-subscriber:1.13.0"` |  |
+| portal.server.graphqlServer.env.LITMUS_CHAOS_OPERATOR_IMAGE | string | `"litmuschaos/chaos-operator:1.13.2"` |  |
+| portal.server.graphqlServer.env.LITMUS_CHAOS_RUNNER_IMAGE | string | `"litmuschaos/chaos-runner:1.13.2"` |  |
```
Comment (rajdas98) [inline charts/litmus-2-0-0-beta/README.md:68]: Can you add EVENT_TRACKER and CHAOS_EXPORTER also?

**Review by rajdas98 — CHANGES_REQUESTED** [review]

**Review by rajdas98 — APPROVED** [review]

**Comment by Jasstkn** [comment]
@ispeakc0de why litmus-portal helm chart was renamed?

**Comment by ksatchit** [comment]
Hi @Jasstkn -- so, as we build towards 2.0 GA via fortnightly beta releases ([2.0.0-beta1](https://github.com/litmuschaos/litmus/releases/tag/2.0.0-Beta1) is the second in the series), there is a need to start messaging litmus as a single entity & not as multiple pieces (operator/portal/kube-chaos exps). 

The portal, when installed, will install the chaos-operator during target registration as well as embed all the publicly available experiments, via MyHub to be able to construct workflows. In other words, this single chart should be sufficient for users to install all the control-plane microservices as well as the experiment artifacts, i.e., we would eventually (ie., until we hit GA 2.0) deprecate the other charts in favor of keeping this single chart. 

Considering the above, we wanted to start referring to the portal chart as litmus-2-0-0-beta into which the versions would be updated w/ beta1/-beta2/-beta3 images until we hit GA. At which time, there would be a new name for the chart. 

This approach would mean creating a fresh chart 2 times - now (portal -> litmus-2-0-0-beta) & once we are ready for GA. This seems less than ideal under normal circumstances, but seems alright considering it is test/beta software that is yet to see significant consumption.

**Comment by Jasstkn** [comment]
> Hi @Jasstkn -- so, as we build towards 2.0 GA via fortnightly beta releases ([2.0.0-beta1](https://github.com/litmuschaos/litmus/releases/tag/2.0.0-Beta1) is the second in the series), there is a need to start messaging litmus as a single entity & not as multiple pieces (operator/portal/kube-chaos exps).
> 
> The portal, when installed, will install the chaos-operator during target registration as well as embed all the publicly available experiments, via MyHub to be able to construct workflows. In other words, this single chart should be sufficient for users to install all the control-plane microservices as well as the experiment artifacts, i.e., we would eventually (ie., until we hit GA 2.0) deprecate the other charts in favor of keeping this single chart. (There is a beta docs site coming up that is trying to provide a user-flow on these lines here: https://litmusdocs-beta.netlify.app/)
> 
> Considering the above, we wanted to start referring to the portal chart as litmus-2-0-0-beta (basically avoid calling it by a "component" name) into which the versions would be updated w/ beta1/-beta2/-beta3 images until we hit GA. At which time, there would be a new name for the chart.
> 
> This approach would mean creating a fresh chart 2 times - now (portal -> litmus-2-0-0-beta) & once we are ready for GA (chart: litmus). This seems less than ideal under normal circumstances, but seems alright considering it is test/beta software that is yet to see significant consumption.

Ok, I wasn't aware of such changes.. And it's kinda surprising that you've decided to delete litmus-portal from chart repo even if it's deprecated. All users can't update its installation properly.

**Comment by ksatchit** [comment]
litmus-portal chart removal was a mistake (it is deprecated, but maybe we should keep it) - we should be putting it back IMO.

**Comment by Jasstkn** [comment]
> litmus-portal chart removal was a mistake (it is deprecated, but we should keep it) - we should be putting it back IMO (i.e., gh-pages)

That's why changes should be reviewed properly...

**Pull request merged**